### PR TITLE
Add max-length prop

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**

### CODE REVIEW

File: yarn-error.log [hunk 1]
```diff
@@ -8612,8 +8612,8 @@ Trace:
   Error: Command failed.
   Exit code: 1
   Command: sh
-  Arguments: -c karma start --single-run
-  Directory: /Users/cambriadeveloper/Desktop/input/react-wood-duck
+  Arguments: -c start-storybook -p 6006
+  Directory: /Users/cambriadeveloper/Desktop/rect/react-wood-duck
```
Comment (NagaPradeepGajula-osi) [inline yarn-error.log]: Is this a typo and why do we need these changes?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-wood-duck",
-  "version": "0.1.52",
+  "version": "0.1.55",
```
Comment (NagaPradeepGajula-osi) [inline package.json]: good catch. Finally update to the current version.

File: stories/components/InputComponent.js [hunk 3]
```diff
@@ -39,6 +39,7 @@ const InputComponentStory = withInfo(
       gridClassName="col-md-12 col-sm-12 col-xs-12"
       labelClassName="Full Name"
       placeholder="Enter Full Name"
+      maxLength={10}
```
Comment (kelfink) [inline stories/components/InputComponent.js]: curls are unnecessary

Comment (Sushmareddy-OSI) [reply stories/components/InputComponent.js]: if not in curly braces, It won't accept numbers

Comment (kelfink) [reply stories/components/InputComponent.js]: I was wrong

**Pull request merged**